doc/pipeline_process.md

```diff
@@ -0,0 +1,80 @@
+# Pipeline process
+
+[Pipeline for QA](https://github.com/status-im/status-mobile/projects/7) is a project board for developers and testers used to track the status of a pull request, get reviews and manual testing, _and run autotests_ (_temporary disabled_).
+
+The generally accepted recommendations for its use are described below:
+
+## Opening a PR
+- Once a PR is created, it moves to the ```REVIEW``` column where a review will be requested automatically.
+- You can also request a review inside the PR from a particular person if needed.
+- When creating a PR, do not forget to assign it to yourself.
+- Also in case the PR adds new functionality, a short description would be appreciated.
```

Comment:
I think it would be good to add a point somewhere in here that some details steps to test the work should be included. sorry if this is somewhere below 😅

Reply:
I mean the developer provides some notes to the QA team on how to test their work.

E.g To test: this pr allows a user to join a community.
Navigate to the community overview page for a community that the user is not a member of. The user should be able to click the join button and then a bottom sheet will show allowing the user to accept the rules and join the community.

After joining the bottom sheet should hide and the initial join button should be hidden from the community overview page.

Something like this :)

Reply:
For this pr: https://github.com/status-im/status-mobile/pull/14874

I have 

```
To test:

go to a 1-1 chat
send a message "Test 1"
pin message "Test 1"
send a message "Test 2"
pin message "Test 2"

expected result - "Test 2" should be shown as the pinned message on screen banner.

Can also test other variants of this to ensure the latest pinned message is showing in the banner
```

Reply:
Perhaps @ilmotta  has some good examples as I've seen on his prs too :)

Reply:
This document isn't really about how the developer should organize the PR, but more about the process between dev and QA. I believe that the information you describe we already have in the draft when creating new PR. Right after creation it contains mock sections that the developer should fill.